packages/babel-plugin/flow_modules/@babel/core/index.js.flow

```diff
@@ -308,7 +308,7 @@ export interface TransformOptions {
    *
    * Default: `[]`
    */
-  presets?: ?$ReadOnlyArray<PluginItem>;
+   presets?: ?$ReadOnlyArray<PluginItem>;
```

Comment:
I think there is an extra space here :P
Maybe it would be a good idea to add this file to the linter or a prettier